src/main/java/beam/router/r5/OsmToMATSim.java

```diff
@@ -33,9 +33,9 @@
     private final static String TAG_ACCESS = "access";
     private final static String[] ALL_TAGS = new String[]{TAG_LANES, TAG_HIGHWAY, TAG_MAXSPEED, TAG_JUNCTION, TAG_ONEWAY, TAG_ACCESS};
     public final Map<String, BEAMHighwayDefaults> highwayDefaults = new HashMap<>();
-    private final Set<String> unknownHighways = new HashSet<String>(); // Used for logging in OsmNetworkReader
-    private final Set<String> unknownMaxspeedTags = new HashSet<String>();
-    private final Set<String> unknownLanesTags = new HashSet<String>();
+    private final Set<String> unknownHighways = new HashSet<>(); // Used for logging in OsmNetworkReader
```

Comment:
Trailing comment is useless. Consider removing it

Reply:
It has some value. I'm all for removing comments in favor of living code - but it seems like it might have been added for a reason - I'd blame and see who added it and ask them first if you do remove it.